File: Algorithmique appliquée/chapitre4/cpp/exercice1.cpp
/* programme factorielles (exercice 1)
*  but : saisir des entiers naturels et afficher leurs factorielles
*  date : 12/12/2016
*/

#include <iostream>
#include <maBiblio.h>

using namespace std;

int main()
{
    unsigned int nb ;
    cout << "Entrez un nombre (0 pour finir) = " ;
    cin >> nb ;
    while (nb != 0) {
        cout << "factorielle = " << factorielle(nb) ;
        cout << endl << "Entrez un nombre (0 pour finir) = " ;
        cin >> nb ;
    }

    return 0;
}

File: Algorithmique appliquée/chapitre4/python/exercice1.py
# -*-coding:utf-8 -*
# Programme exercice 1
# saisir des entiers naturels et afficher leurs factorielles
# date : 12/12/2016

import os
import sys
sys.path.append("..\\bibliotheque")
from maBiblio import *


nb = int(input("Entrez un nombre (0 pour finir) = "))
while nb != 0 :
    print("factorielle = ", factorielle(nb))
    nb = int(input("Entrez un nombre (0 pour finir) = "))
    

os.system("pause")
